docs/src/Layer.doc.js

```diff
@@ -42,6 +42,62 @@ card(
   />
 );
 
+card(
+  <Example
+    description="
+    Since Layer is rendered within a portal element being manually added to the DOM, it is supposed to be rendered and dismissed only once. 
+    Any re-renders triggered by the parent will effectively remove the portal element from the DOM, hence it won't be re-created and internal elements like Sheet may not reappear.
+    The example below demonstrate a technique to avoid Layer re-rendering by making sure parent re-renders don't force re-renders by passing useCallback function props and preventing re-renders with memo().
```

Comment:
typo: demonstrates

Also: it's a bit of a run-on sentence, and could use some clarification. Consider breaking into smaller chunks, something like:

"The example below demonstrates a technique to avoid Layer re-rendering by making sure parent re-renders don't force children re-renders. This is accomplished by memoizing locally-defined functions with React.useCallback if they're passed down as props, and memoizing the entire component with React.memo()."

Reply:
Suggestion accepted, thank you!